### PR TITLE
Add Carlo Varni to the project

### PR DESCRIPTION
**Pull request by CarloVarni**
New PostDoc Researcher at UC Berkeley
PI: Heather Gray

### CODE REVIEW

**Comment by matthewfeickert** [comment]
Welcome @CarloVarni! :) Great to be working with you again.

File: _data/people/cvarni.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name: Carlo Varni
+title:
```
Comment (matthewfeickert) [inline _data/people/cvarni.yml]: @CarloVarni can you also add your title at UC Berkeley and your email? 

c.f. https://github.com/iris-hep/iris-hep.github.io-source/blob/f6e4d0202554c74b3fa661b0e54c711486b088f2/_data/people/matthewfeickert.yml#L1-L8

as an example.

**Review by matthewfeickert — CHANGES_REQUESTED** [review]

File: _data/people/cvarni.yml (hunk 1 above)
Comment (Psicognomo) [reply _data/people/cvarni.yml]: Hi @matthewfeickert, nice to see you again! 
I've added both the title and the e-mail

**Review by matthewfeickert — APPROVED** [review]
LGTM.

File: _data/people/cvarni.yml (hunk 1 above)
Comment (CarloVarni) [reply _data/people/cvarni.yml]: Hi @matthewfeickert, nice to see you again. I have added both the title and the Mail.

**Comment by henryiii** [comment]
FYI, the shortname seems to need to match the filename, I had to rename the file in #674 as this was making a random blank card. I've added a check for blank cards. Not sure why we need to have a shortname if the filename already provides that info, might look into removing it eventually.

**Comment by matthewfeickert** [comment]
Ah sorry about that @henryiii. Though thanks for mentioning that as I'll be sure to look for that in the future.

**Comment by henryiii** [comment]
It should kill the build from now on instead of making an empty card. :)

**Pull request merged**